lib/mpl_toolkits/mplot3d/tests/test_axes3d.py

```diff
@@ -1766,6 +1774,51 @@ def test_shared_axes_retick():
     assert ax2.get_zlim() == (-0.5, 2.5)
 
 
+def test_rotate():
+    """Test rotating using the left mouse button."""
+    fig = plt.figure()
+    ax = fig.add_subplot(1, 1, 1, projection='3d')
+    ax.view_init(0, 0, 0)
+    ax.figure.canvas.draw()
+
+    # drag mouse horizontally to change azimuth
+    dx = 0.1
+    dy = 0.2
+    ax._button_press(
+        mock_event(ax, button=MouseButton.LEFT, xdata=0, ydata=0))
+    ax._on_move(
+        mock_event(ax, button=MouseButton.LEFT,
+                       xdata=dx*ax._pseudo_w, ydata=dy*ax._pseudo_h))
+    ax.figure.canvas.draw()
+    print(ax._pseudo_w)
+    print(ax.elev)
+    assert ax.azim == -dx*180
+    assert ax.elev == -dy*180
+    assert ax.roll == 0
+
+
+def test_rotate_roll():
```

Comment:
I think this test looks great! I would suggest that you combine both tests into a single function, since the only code different is the roll angle.